modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-synchronization_{context}"]
+= Security context constraint synchronization with pod security standards
+
+{product-title} implements link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+In addition to the global pod security admission control configuration, a pod admission controller applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
+
+The controller examines `ServiceAccount` object permissions to apply pod security profiles constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission `warn` and `audit` labels are set to the most privileged pod security profile found in the namespace to prevent warnings and audit logging as pods are created.
+
+Namespace labeling is based on consideration of namespace-local service account privileges.
+
+Applying pods directly might use the SCC privileges of the user who runs the pod. However, user privileges are not considered during automatic labeling.
+
+[IMPORTANT]
+====
+SCC synchronization is disabled by default for user-created namespaces that are named with the prefix `openshift-`. Namespaces that have this prefix are typically system namespaces; by convention, a controller exists to manage them. 
```

Comment:
```suggestion
SCC synchronization is disabled by default for namespaces that are named with the prefix `openshift-`. Namespaces that have this prefix are typically system namespaces; by convention, a controller exists to manage them. 
```

What do you mean by "user-created"? I understand that if the namespace is prefixed with `openshift` then they will not be synced by default.

Reply:
User-created means not created by OpenShift.

TODO (me): propose an improvement to this part of the docs

Reply:
> Also, we need to clarify here that those are not enforced as restricted for now but will be in the next release.

Perhaps in the release notes, not here.

Reply:
> What do you mean by "user-created"? I understand that if the namespace is prefixed with openshift then they will not be synced by default.

```
Namespaces with an "openshift-" name prefix that were not created by the system during the installation are not being synchronized by default.
```
Maybe something like this?